content/v2.0/account-management/multi-user/remove-user.md

```diff
@@ -0,0 +1,17 @@
+---
+title: Remove a user
+list_title: Remove a user
+description: >
+  Remove a user from an organization
+weight: 103
+menu:
+  v2_0:
+    parent: Manage multiple users
+    name: Remove a user
+---
+
+## Remove a user from {{< cloud-name >}}
```

Comment:
Curious why not use "delete" ?

Reply:
"Delete" implies that the user is completely removed from the system. They aren't. They just don't have access to your organization, but they may belong to other organizations.

Reply:
Does the UI say delete? Perhaps rename procedure "Delete user from organization"?
In context of an organization, delete and remove imply the same. How many users are going to be part of multiple orgs?